### ISSUE TITLE
PyPI (and name)

### ISSUE THREAD

**Issue by joezuntz** [jlvdb/hyperbolic#1]
Since this is now a RAIL dependency could we have it added to PyPI?

To do that you'd have to change its name, since there's already a package called hyperbolic.  How about `luptitudes`?

**Comment by jlvdb** [comment]
Good point, this has been on my agenda as well. I'll think about a good name and update RAIL once it is on PyPI.
